simpeg/data.py

```diff
@@ -1,9 +1,18 @@
+import sys
 import numpy as np
 import warnings
 
 from .survey import BaseSurvey
 from .utils import mkvc, validate_ndarray_with_shape, validate_float, validate_type
 
+if sys.version_info >= (3, 13):
+    # warnings.deprecated available in Python >=3.13
+    from warnings import deprecated
```

Comment:
You could instead just use a try… except 
ImportError block here too instead of comparing versions